src/java/README.md

```diff
@@ -16,6 +16,7 @@ Installs Java, SDKMAN! (if not installed), and needed dependencies.
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
 | version | Select or enter a Java version to install | string | latest |
+| additionalVersions | Comma separated list of additional versions to install | string | empty |
```

Comment:
We need to expose `additionalVersions` as a Feature option.

Reply:
it's already exposed it seems. Pls do check the tests written for it in the scenario - `install_additional_java`

Reply:
The option is created, but it is not listed in https://github.com/devcontainers/features/blob/main/src/java/devcontainer-feature.json

Hence, it won't be exposed to other IDEs. Instead of directly updating the README, we'd need to update https://github.com/devcontainers/features/blob/main/src/java/devcontainer-feature.json which publicizes the option.

Reply:
thanks for elaborating